runtime/onert/python/api/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+# CMakeLists.txt
+
+# refer to https://github.com/Samsung/ONE/issues/11368
+
+if(CMAKE_CROSSCOMPILING)
+    set(CMAKE_FIND_ROOT_PATH_MODE_PACKAGE ONLY "Set to find packages only in cross root path by force")
+endif()
+
+find_package(Python COMPONENTS Interpreter Development)
+
+# Tell pybind11 where the target python installation is
+set(PYTHON_INCLUDE_DIRS ${Python_INCLUDE_DIRS} CACHE INTERNAL "Cross python include path")
+set(PYTHON_LIBRARIES ${Python_LIBRARIES} CACHE INTERNAL "Cross python include path")
+set(PYTHON_MODULE_PREFIX "lib" CACHE INTERNAL "Cross python lib prefix")
+set(PYTHON_MODULE_EXTENSION ".so" CACHE INTERNAL "Cross python lib extension")
+
+# Disable pybind11 python search mechanism
+set(PYTHONLIBS_FOUND TRUE CACHE INTERNAL "")
```

Comment:
I'm not sure these lines are necessary. These variables were required by pybind11's cmake files. Did the current cmake file here also require these variables?

Reply:
We thought it was unnecessary and tried to do a cross build, but when creating a modularized shared library using pybind11.

`add_library(nnfw_api_pybind MODULE ${NNFW_API_PYBIND_SOURCES})`

 the problem of not being able to find the python path continued to occur, and the code could not be removed.